### PR TITLE
feat(scan): add `plustek-sdk` for precinct scanner

### PR DESCRIPTION
**Pull request by eventualbuddha**
This is ready for initial review, but still has some tidying up to do before merge:

- [x] add CI config
- [ ] more docs for RPC/IPC format
- [x] better naming for things?
- [x] full test coverage

### CODE REVIEW

File: libs/plustek-sdk/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "plustek-sdk",
+  "version": "1.0.0",
+  "description": "Wraps the official plustek SDK for use with the VTM 300 via NodeJS.",
+  "keywords": [],
+  "license": "GPL-3.0",
```
Comment (benadida) [inline libs/plustek-sdk/package.json:6]: AGPL ?

File: libs/plustek-sdk/src/config.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  savepath?: string
+  'filename-format'?: string
+  'filename-beginindex'?: number
+  'paper-size'?:
+    | 'A3'
+    | 'A4'
+    | 'A5'
+    | 'B5'
+    | 'B6'
+    | 'Letter'
```
Comment (benadida) [inline libs/plustek-sdk/src/config.ts:11]: + 'Legal' ?

**Review by benadida — APPROVED** [review]
this is great.

File: libs/plustek-sdk/src/config.ts (hunk 2 above)
Comment (eventualbuddha) [reply libs/plustek-sdk/src/config.ts:11]: These are all that were listed in the official SDK. However, the paper size selected seems to have no effect as it'll scan Letter or Legal with "Letter" size selected.

**Pull request merged**